projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/__mocks__/index.ts

```diff
@@ -0,0 +1,130 @@
+import { PartyMessages } from '../domain';
+
+export const partyMessagesMockData: PartyMessages[] = [
+  {
+    partyName: 'John Smith - Appellant',
+    roleOnCase: null,
+    partyMessages: [
+      {
+        id: 'case-message-001',
+        subject: 'Review attached document',
+        body: 'Please review attached document and advise if hearing should proceed?',
+        attachments: [],
+        isHearingRelated: true,
+        hearingDate: new Date(2023, 0, 10),
```

Comment:
Dates will come from the API in string format, not as a date object

Reply:
I have changed the data type from date to string.